omc3/tune_analysis/fitting_tools.py

```diff
@@ -44,11 +51,131 @@ def do_odr(x, y, xerr, yerr, order):
     Returns: Odr fit. Betas order is index = coefficient of same order.
     """
     LOG.debug("Starting ODR fit.")
-    fit_np = np.polyfit(x, y, order)[::-1]  # using polyfit as starting parameters
+
+    # Poly-Fit for starting point ---
+    fit_np = np.polyfit(x, y, order)[::-1]  # polyfit order is inversed to poly_func
```

Comment:
If this isn't too much work, maybe we could use `Polynomial.fit` as `numpy` recommends for new code: https://numpy.org/doc/stable/reference/routines.polynomials.html

Reply:
Order of returned coefficients seems opposite though. So the [::-1] from here would go away